source/portable/mbedtls/core_pkcs11_mbedtls.c

```diff
@@ -45,6 +45,8 @@
 #include "mbedtls/sha256.h"
 #include "mbedtls/platform.h"
 #include "mbedtls/threading.h"
+#include "mbedtls/cipher.h"
```

Comment:
Not a bad include per-say, but I don't think it is necessary, since `cmac.h` will pull this in?